internal/checker/environment.go

```diff
@@ -0,0 +1,164 @@
+package checker
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/avorty/spito/pkg/shared"
+	"github.com/avorty/spito/pkg/vrct/vrctFs"
+	"os"
+	"path/filepath"
+)
+
+var EnvironmentDataPath = filepath.Join(shared.UserHomeDir, ".local/state/spito/environment-data.json")
```

Comment:
Can you add `.local/state/sptio` to env or make it global const?

Reply:
I cannot make it neither constant nor build env because it is different based on which user is running spito

Reply:
But I mean only `.local/state/spito` as it's repeated sometimes

Reply:
Okay, but I would rather make constant value instead of putting it into `buil.env`
Is it okay?

Reply:
ok